vendor/github.com/zorkian/go-datadog-api/monitors.go

```diff
@@ -137,6 +137,18 @@ type Creator struct {
 	Name   *string `json:"name,omitempty"`
 }
 
+// MuteMonitorScope specifies which scope to mute and when to end the mute
+type MuteMonitorScope struct {
+	Scope *string `json:"scope,omitempty"`
+	End   *int    `json:"end,omitempty"`
+}
+
+// UnmuteMonitorScopes specifies which scope(s) to unmute
+type UnmuteMonitorScopes struct {
+	Scope     *string `json:"scope,omitempty"`
+	AllScopes *bool   `json:"all_scopes,omitempty"`
```

Comment:
Would it make more sense to just have `Scopes` rather than `AllScopes`?

Reply:
This is really just mimicking the parameters that are available through DataDog API: https://docs.datadoghq.com/api/?lang=python#unmute-a-monitor

So I'd like to leave it as it is.